assets/js/components/notifications/CTA.js

```diff
@@ -51,7 +53,10 @@ const CTA = ( {
 				{ description }
 			</div>
 		) }
-		{ ctaLabel && (
+		{ ctaLabel && ctaType === 'button' && (
+			<Button onClick={ onClick }>{ ctaLabel }</Button>
+		) }
+		{ ctaLabel && ctaType === 'link' && (
```

Comment:
Let's keep using the existing condition where currently the `Link` is printed. The `Button` component largely supports the same props (it can also be used for a semantic link that _looks_ like a button).

For example, we could do something like this:
```js
const Tag = ctaType === 'button' ? Button : Link;
```

And then render that via `<Tag ...` instead of the hard-coded `<Link ...` below.

Reply:
@felixarntz, we get warnings for the invalid props passed to the `Button` component. IMO, we could keep it separate to avoid these warnings. @tofumatt, WDYT?

![Screenshot 2022-06-15 at 11 15 35 AM](https://user-images.githubusercontent.com/24408261/173751902-7c4bc664-7680-4b6c-9d7b-e90284b219dd.png)

Reply:
Based on the type you could pass in the correct args, eg. assemble the relevant props by doing something like:

```js
const ctaProps = {};

if (ctaType === 'button') {
  ctaProps.onClick = onClick;
  ctaProps.href = ctaLink;
}

if (ctaType === 'link') {
  ctaProps.href = ctaLink;
  ctaProps.hideExternalIndicator = ctaLinkExternal;
}
```

etc.

That said I think this is a more roundabout approach, I'd prefer just outputting the components separately as is done here, because then all the props and how they map to each component is more explicit.

That said, at least the `aria-label` and `href` props seem like they should be passed to `Button` as it should accept them.

Reply:
Honestly, I prefer to keep the components separately and pass the props explicitly instead of passing them as 
`{ ...ctaProps }`. Let me pass the `aria-label` and `href` props to the `Button` component.